tests.py

```diff
@@ -102,7 +103,7 @@ def test_events():
     # - extracts the system call
     # - strips durations if present
     # - assembles system calls split across several lines with <unfinished...> <... resumed>
-    log_lines = [
+    log_lines = StringIO('\n'.join([
```

Comment:
Why not use `Iterable[str]` instead of `TextIO` in the signature of `events()` and keep the test simple?